sklearn/tests/test_docstring_parameters.py

```diff
@@ -235,18 +235,6 @@ def test_fit_docstring_attributes(name, Estimator):
         # default="auto" raises an error with the shape of `X`
         est.set_params(n_components=2)
 
-    # FIXME: TO BE REMOVED in 1.4 (avoid FutureWarning)
```

Comment:
Just checking: I guess there was there a mistake in the comment here.

Reply:
yes because now they have normalize=False by default and leaving the default does not raise a FutureWarning, so this can be removed right away